test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/TagHelperOutputExtensionsTest.cs

```diff
@@ -979,8 +979,8 @@ public bool Equals(TagHelperAttribute attributeX, TagHelperAttribute attributeY)
                 // Normal comparer (TagHelperAttribute.Equals()) doesn't care about the Name case, in tests we do.
```

Comment:
If this class really must exist, same comment re caring about `Name` case in `GetHashCode()` as in the Razor PR.